automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/summary/WorkspaceSummaryPage.scala

```diff
@@ -233,7 +153,7 @@ class WorkspaceSummaryPage(namespace: String, name: String)(implicit webDriver:
     await ready new CloneWorkspaceModal
   }
 
-  private def isEditing: Boolean = {
+  private[summary] def isEditing: Boolean = {
     !sidebar.editButton.isVisible
```

Comment:
It'd be a better verification if this is checking for `sidebar.cancelButton` and `saveButton` visible. Correctly verifying expected UI elements displayed.